exp/anbox-cloud.md

```diff
@@ -31,62 +31,78 @@ See the following table for a comparison of features for the different variants:
 
-For example, a simple Android application testing service would provide a user-facing interface dealing with things like authentication and user management, and would communicate with the REST API to add applications or start and stop containers when a user asks to.
+Users can connect to AMS via CLI or API calls on port 8444. AMS is installed as a snap on each of the control nodes and interacts with Anbox containers, requesting and releasing resources as per demand. 
 
-Anbox Cloud can be heavily customised and extended via [**platform plugins**](https://discourse.ubuntu.com/t/anbox-cloud-sdks/17844#anbox-platform-sdk) and [**addons**](https://discourse.ubuntu.com/t/managing-addons/17759). Platform plugins and addons can be built to add specific streaming capabilities, perform operations within Android containers and much more. One example of a platform plugin is the [**Anbox WebRTC Platform**](https://discourse.ubuntu.com/t/anbox-platforms/18733) used in the Anbox Streaming Stack. Addons are ways to customise the base image by installing additional software and running scripts on different life cycle hooks.
+**Anbox Management Client** - You can choose to install AMC on other machines to [control AMS remotely](https://discourse.ubuntu.com/t/how-to-control-ams-remotely/17774), but it is generally installed together with AMS. A developer or system administrator will manage AMS through the command line interface (AMC) or through custom-built tools interacting with the AMS HTTP API.
+      
+**etcd** - etcd is the database that is used to store the data managed by AMS. It provides a reliable way to store data across a cluster of machines. It gracefully handles primary node selections during network partitions and will tolerate machine failure.
 
-### Streaming stack
+The core stack also contains the Anbox subcluster which is the central part of an Anbox Cloud deployment. This subcluster is also called “region”, which indicates that you would usually have one subcluster per geographic region that you want to serve. Each Anbox subcluster requires a machine that hosts the management layer of Anbox Cloud.
+
+Each Anbox subcluster also has a number of LXD worker nodes that form a LXD cluster. Each LXD worker node runs the following components:
 
-Starting from 1.4, Anbox Cloud comes with an easy to use streaming solution. The [**Anbox Streaming Stack**](https://discourse.ubuntu.com/t/streaming-android-applications/17769) is a collection of components designed to run containers on GPU-equipped machines and stream their visual output to clients via [WebRTC](https://webrtc.org/).
+  **LXD** - The LXD daemon spawns a number of LXD containers containing Anbox related configuration. These containers provide an Ubuntu environment that uses the hardware of the LXD worker node, including CPUs, disks, networks and if available, GPUs. Each LXD/Anbox container runs exactly one Android container, that the end user can access/stream. 
```

Comment:
Yes, we did have a discussion about that. So we are calling them LXD containers or LXD images since they are essentially that but have Anbox configuration. So when it gets confusing or if we are setting the context, we can explain it as in the first statement - 'LXD containers containing Anbox-related configuration'.

In any case, I am adding a note here clarifying what we mean.